.github/workflows/windows-visual-studio-cminpack-install.yaml

```diff
@@ -3,35 +3,141 @@ run-name: MSVC - Build, test and install cminpack on Windows through CMake
 on: [push, pull_request]
 
 jobs:
+  build-BLAS:
```

Comment:
why not openblas? Who would use the reference BLAS implementation? the following figure is from [Wittwer 2008](https://www.wittwer.nl/wp-content/uploads/2009/08/blas_lapack.pdf), and OpenBLAS is based on GotoBLAS.
![image](https://github.com/devernay/cminpack/assets/3778071/ee72acfa-389c-4c90-aacd-2e7e12d1ade5)